src/common/compat/fcntl.hpp

```diff
@@ -22,6 +21,7 @@ typedef off64_t loff_t;
 #endif
 
 #ifdef __linux__
+typedef off_t off64_t;
```

Comment:
Won't `off_t` be 4 bytes on 32-bit platforms?

Reply:
Assumption is the `-D_FILE_OFFSET_BITS=64` is always passed to build.